### PR TITLE
Task#14: Capacity testing pipeline in GOCD

### PR DESCRIPTION
**Pull request by GioBlanco**
Note: I also added the dynamic responce to GIT_COMMIT_HASH environment variable.
2023-06-12_19h02_24
2023-06-12_18h18_20
2023-06-12_19h12_40

### CODE REVIEW

File: Test1.gb.gocd.yaml [hunk 1]
```diff
@@ -131,4 +145,48 @@ pipelines:
...
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      GIT_COMMIT_HASH: 1.2
+      TARGET_HOST: 192.168.0.46
+    materials:
+      Test1:
+        pipeline: Test1
+        stage: PackageAndPublish
```
Comment (vcarmen) [inline Test1.gb.gocd.yaml]: It should depend on the Previous pipeline (Acceptance.) if acceptance passes, then continue with the Capacity testing

File: Test1.gb.gocd.yaml [hunk 2]
```diff
@@ -131,4 +145,48 @@ pipelines:
...
+                pipeline: Test1
+                stage: PackageAndPublish
+                job: docker
+                source: docker-compose.gb.yaml
+                is_file: true
+                run_if: passed
+            - script: |
+                export GIT_COMMIT_HASH=$(cat gitHash.txt)
+                export TAG_VERSION=${GIT_COMMIT_HASH} 
+                DOCKER_HOST=ssh://$TARGET_HOST docker-compose -f docker-compose.gb.yaml up -d
```
Comment (vcarmen) [inline Test1.gb.gocd.yaml]: Add stage to run capacity testing with a step: command here to run capacity testing

**Review by vcarmen — CHANGES_REQUESTED** [review]

File: Test1.gb.gocd.yaml (hunk 1 above)
Comment (GioBlanco) [reply Test1.gb.gocd.yaml]: I completed the modification
2023-06-13_11h11_28

File: Test1.gb.gocd.yaml (hunk 2 above)
Comment (GioBlanco) [reply Test1.gb.gocd.yaml]: I added thte stage
2023-06-13_21h39_01

**Review by vcarmen — CHANGES_REQUESTED** [review]
Update the dependency of Capacity Pipeline: It should fetch from Acceptance pipeline

**Review by vcarmen — APPROVED** [review]

**Pull request merged**